mono/metadata/exception.h

```diff
@@ -5,6 +5,7 @@
 #ifndef _MONO_METADATA_EXCEPTION_H_
 #define _MONO_METADATA_EXCEPTION_H_
 
+#include <mono/metadata/object-forward.h>
```

Comment:
This will break the embedding API: `exception.h` is an installed header but `object-forward.h` isn't.

https://github.com/mono/mono/blob/e43adf3cd5a9bf6ba8c9ca2bef00b266361cd5bc/mono/metadata/Makefile.am#L333-L358

Reply:
Right, should be fixed by 3b892d45b000f7e9d4ec988dd00beebede1afe9d.